pkg/collector/corechecks/sbom/check.go

```diff
@@ -180,7 +181,8 @@ func (c *Check) Run() error {
 	)
 
 	// Trigger an initial scan on host
-	c.processor.processHostRefresh()
+	hostSbomChan := make(chan sbom.ScanResult, 1)
```

Comment:
since you create a new chan, who is responsible for closing it? 
if it is not a `defer` in this function could you add a comment to explains who is responsible of closing it 🙇

Reply:
Added a comment on it. I think we don't need to close it. Actually go doesn't enforce closing a channel.

Reply:
Normally, in that case, the SBOM scanner will try to send and fail without blocking. In that case it should just log an error and free the channel. I think the scanner should be the one doing it